### PR TITLE
master-lps-101038

### PR DESCRIPTION
**Pull request by liuxingye**
https://issues.liferay.com/browse/LPS-116322

### CODE REVIEW

**Comment by liuxingye** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Checkbox.path [hunk 1]
```diff
@@ -88,6 +88,11 @@
 	<td>//label[contains(.,'${key_checkboxName}')]/input[@type='checkbox' and contains(@class,'disabled')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>CHECKBOX_DISABLED_EXPORT_TRANSLATION</td>
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Checkbox.path:92]: Can we refactor this path and CHECKBOX_DISABLED and remove "CHECKBOX"? That's already mentioned in the test file. So the locator should be Checkbox#DISABLED_EXPORT_TRANSLATION

File: modules/apps/translation/translation-test/src/testFunctional/tests/Translations.testcase [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+		var translationZipFileName = TestCase.getDownloadedTempFileName(fileNamePattern = "Web Content Title*.zip");
+
+		echo("## translationZipFileName: ${translationZipFileName}");
+
+		if (contains("${translationZipFileName}", "en_US")) {
+			echo("This is a exported translation file.");
+		}
+		else {
+			fail("FAIL!");
+		}
```
Comment (austinchiang) [inline modules/apps/translation/translation-test/src/testFunctional/tests/Translations.testcase:80]: Can we make this into a reusable macro?

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Checkbox.path (hunk 1 above)
Comment (liuxingye) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Checkbox.path:92]: see https://github.com/austinchiang/liferay-portal/pull/831

File: modules/apps/translation/translation-test/src/testFunctional/tests/Translations.testcase (hunk 2 above)
Comment (liuxingye) [reply modules/apps/translation/translation-test/src/testFunctional/tests/Translations.testcase:80]: See https://github.com/austinchiang/liferay-portal/pull/831

**Pull request closed without merging**